### ISSUE TITLE
[Bug]

### ISSUE THREAD

**Issue by c3founder** [windingwind/zotero-better-notes#287]
When I press the citation button (below image) I get undefined printed in the note:

![image](https://user-images.githubusercontent.com/70454003/208066574-963e69cd-0823-421a-a202-3fbde60becd7.png)

Also, I don't see the mentioned option in the menu here:
https://github.com/windingwind/zotero-better-notes/wiki/4.7-Quick-Citation-Insertion-Quick-Cite.en

Thanks!

**Comment by windingwind** [comment]
Please check the plugin version.

**Comment by c3founder** [comment]
It is the latest version. Maybe I'm doing something wrong?

**Comment by windingwind** [comment]
The citation style shows up when you are in the workspace tab/window. This might be a bug though;

Could you please check if it's correctly set?

**Comment by gtyysq** [comment]
我也出现了`插入引文`后得到"undefined"的情况。
但是原来`插入引文`是能正常插入的，直到我换了一下`笔记引用格式`
<img width="769" alt="截屏2023-01-10 12 53 16" src="https://user-images.githubusercontent.com/59784122/211465628-f34c3e57-d62b-4e09-b8c3-6652b559b70d.png">
之后再`插入引文`就一直是“undefined”的情况了
<img width="176" alt="截屏2023-01-10 12 53 57" src="https://user-images.githubusercontent.com/59784122/211465835-65467bb2-c5e0-40e2-8827-a9a1ddd1951b.png">
而且`编辑`菜单里的`笔记引用格式`选项也消失了
<img width="278" alt="截屏2023-01-10 12 54 17" src="https://user-images.githubusercontent.com/59784122/211466086-d360ef6e-e3a9-4513-85b9-91799d396c7c.png">

**Comment by hyunsung-lee** [comment]
I had this problem too, and I discovered that the menu entry for selecting the citation style only appears after clicking on the **'Workspace' tab**. It does not appear when you've already clicked into the editing space of the main note. 

![click](https://user-images.githubusercontent.com/79105322/222552869-377edf5f-a9ba-4201-9078-d4402b9b3ab2.png)

After that, you get to choose your preferred citation style. It was not chosen by default, so I guess it was really 'undefined.' It might be better if this had a default value for anyone who doesn't really care which citation style to use in their notes.

![capture](https://user-images.githubusercontent.com/79105322/222552381-bacb1eb2-0cd0-4d66-a09a-337220d7226e.png)

After I selected the citation style, I am not seeing this error.

BTW, amazing plugin. I always dreamt of something like this plugin. Thanks a lot for developing and sharing!
